src/dlgRoomExits.cpp

```diff
@@ -541,7 +541,7 @@ void dlgRoomExits::save()
     exitKey = QStringLiteral("s");
     dirCode = DIR_SOUTH;
     if (s->isEnabled() && !s->text().isEmpty() && mpHost->mpMap->mpRoomDB->getRoom(s->text().toInt()) != nullptr) {
-        if (originalExits.value(dirCode)->destination != s->text().toInt()) {
+        if (originalExits.contains(dirCode) && originalExits.value(dirCode)->destination != s->text().toInt()) {
```

Comment:
Actually this fix will be needed for the other 11 instances...

Reply:
Yeah it is, smurfix could you add it in? Then it's good for a merge.

Reply:
Pushed, for now.

As soon as this is merged I'll check about refactoring those pieces. Should have been done by whoever wrote it in the first place 😒 but oh well.

Reply:
Well it works, the user are happy maybe not the best but improvements are always welcome!